practitioner-service/src/main/java/uk/ac/ox/ndph/mts/practitioner_service/repository/HapiFhirRepository.java

```diff
@@ -64,9 +66,8 @@ public String savePractitionerRole(PractitionerRole practitionerRole) {
         Bundle responseBundle;
         try {
             responseBundle = fhirContextWrapper.executeTransaction(bundle(practitionerRole));
-        } catch (BaseServerResponseException e) {
-            logger.warn(FhirRepo.UPDATE_ERROR.message(), e);
```

Comment:
Is this enum still in use? If not, please remove it.

Reply:
Enum removed.